magnum-zio/src/main/scala/com/augustnagro/magnum/magzio/Transactor.scala

```diff
@@ -1,36 +1,32 @@
 package com.augustnagro.magnum.magzio
 
-import com.augustnagro.magnum.{DbCon, DbTx, SqlException, SqlLogger}
+import com.augustnagro.magnum.{
+  DbCon,
+  DbTx,
+  SqlException,
+  SqlLogger,
+  TransactorOps
+}
 import zio.{Semaphore, Task, Trace, UIO, ULayer, ZIO, ZLayer}
 
 import java.sql.Connection
 import javax.sql.DataSource
 import scala.util.control.NonFatal
 
-class Transactor private (
-    dataSource: DataSource,
-    sqlLogger: SqlLogger,
-    connectionConfig: Connection => Unit,
-    semaphore: Option[Semaphore]
-):
+final case class Transactor(
```

Comment:
IMO `case class` represent data structures. This isn't a data structure so we don't need `case` here
```suggestion
final class Transactor(
```

Reply:
Yeah, I am thinking for a long time to change this documentation. I should have done it before.

Reply:
IMO This is not a big issue at all - I am happy to use ordinary classes in cases like this.